### DISCUSSION TITLE
Can't get exe to run?

### DISCUSSION THREAD

**Discussion by chronomac** [quietvoid/dovi_tool#71] [Q&A] [answered]
I downloaded the latest release, unpacked the tar.gz file and then tried to run the .exe file. That didn't work so I tried the listed commands in Powershell and I simply get back `The command dovi_tool was not found` and `The command dovi_tool.exe was not found`. I know I'm doing something wrong here but unsure what.

**Comment by quietvoid** [comment, marked as answer]
On Windows, you need to run it like this `.\dovi_tool.exe`, and it has to be in your current directory.

**Reply by chronomac** [reply]
That was super fast, and it helped. Thanks. Another question: if I have an MKV file that includes Dolby Vision Profile 7 metadata and I want to convert that to Profile 8, so I can use it on an Apple TV theoretically, then the easiest way to do that would be with the `ffmpeg -i input.mkv -c:v copy -vbsf hevc_mp4toannexb -f hevc - | dovi_tool -m 2 convert --discard -` command, right?

**Reply by quietvoid** [reply]
Yes, although the AppleTV 4K only supports actual profile 5 correctly.

**Reply by chronomac** [reply]
Ah, I thought it had support for 8, since the new iPhones shoot with that profile. Maybe they're adding it later?

**Reply by quietvoid** [reply]
They support profile 8.4, which is HLG.
Profile 7 files are HDR10, and dovi_tool only supports converting them to profile 8.1.

**Reply by chronomac** [reply]
That’s disappointing but thank you for clarifying. So I guess I’m out of luck then, when it comes to keeping the DV metadata and playing back the file on my Apple TV then.

**Reply by quietvoid** [reply]
It'll play in Dolby Vision, but as far as I know the metadata is not used at all.

**Reply by chronomac** [reply]
> > It'll play in Dolby Vision, but as far as I know the metadata is not used at all.
> 
> Reshaper on HLG is being applied in mpv :)

I’m not sure what this means. Can you explain, in layman’s terms?

**Reply by saindriches** [reply]
AFAIK Apple or Dolby itself (in newer applications) does not follow reshaping metadata to convert P8.4 from HLG to PQ, it just provides compatibility for old Dolby Vision devices or software decoder. There is a standalone application in Dolby Encoding Engine for convert encoded P8.4 to SDR, it works fine even without reshaping part in RPU.

**Reply by saindriches** [reply]
Maybe we need move to another place for discussion.

In short, P8.4 is PQ1000 with HLG base layer. While HLG may captured from scene light, It's irrelevant.

When decoded as DV, it is converted to PQ. Mapping is to predict the required signal from encoded signal, but for HLG to PQ, we have an accurate way to get result. 
Then the same processing as other DV profile 8 is applied. It has L1 (and many other blocks for DM4 ver), which affects conversion to SDR, as well as other targets. Original HLG EOTF with adaptive display brightness, or SDR compatibility of HLG is not the way P8.4 work.

**Reply by ghost** [reply]
How I have to run it on MacOs?
